presto-common/src/main/java/com/facebook/presto/common/type/Uuids.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.common.type;
+
+import java.nio.ByteBuffer;
+import java.nio.ByteOrder;
+
+public class Uuids
```

Comment:
Do we as a project have a strong opinion about UUID vs Uuid in class and method names? That is, do we capitalize acronyms like the JDK (URL) or like Google (Url)?

Reply:
I found within the Presto codebase we don't use full capitalization in most cases. Additionally we call the UUID type class `UuidType` so I opted for consistency with the existing naming.